API.md

```diff
@@ -652,7 +652,7 @@ For example, use `is: Joi.number().required()` to guarantee that a joi reference
 
 ```js
 const schema = {
-    a: Joi.any().valid('x').when('b', { is: 5, then: Joi.valid('y'), otherwise: Joi.valid('z') }),
```

Comment:
Nah I'm good with that explanation. Was more curious than anything else :) Thanks!